temporian/core/data/duration.py

```diff
@@ -36,3 +36,74 @@ def hours(value: float) -> Duration:
 
 def days(value: float) -> Duration:
     return value * 60 * 60 * 24
+
+
+def weeks(value: float) -> Duration:
+    return value * 60 * 60 * 24 * 7
+
+
+# Could be 29, 30 or 31 days

```

Comment:
Since a month duration is not accurate, I would not expose such method.
Same for years.